crates/polars-core/src/datatypes/dtype.rs

```diff
@@ -166,6 +166,16 @@ impl DataType {
         self.is_float() || self.is_integer()
     }
 
+    /// Check if this [`DataType`] is a Decimal type (of any scale/precision).
+    pub fn is_decimal(&self) -> bool {
+        #[allow(clippy::match_like_matches_macro)]
```

Comment:
Why not just use the `matches!` macro here?

Reply:
Didn't play well with the requirement to use `feature = "dtype-decimal"` (unless some syntax nicety is available that I'm unaware of?), but I'm not sure how that `#[allow(clippy::match_like_matches_macro)]` ended up there - removing!